app/dashboard/views.py

```diff
@@ -3939,24 +3943,24 @@ def hackathon_save_project(request):
                 'logo': request.FILES.get('logo', project.first().logo)
             })
 
-            project.update(**kwargs)
+            # project.update(**kwargs)
 
-            try:
-                bounty_profile = Profile.objects.get(handle=project.bounty.bounty_owner_github_username.lower())
-                if bounty_profile.chat_id is '' or bounty_profile.chat_id is None:
-                    created, bounty_profile = associate_chat_to_profile(bounty_profile)
+            #try:
```

Comment:
we are removing the chat associated actions?

Reply:
Thanks, i thought i restored these lines. The chat related actions caused crash my environment so i commented this piece of code.

Reply:
@zoek1 there are some docs for the Gitcoin Chat setup, the container will spin up automatically.

If you go through that setup then you'll stop getting any connection issues.

Reply:
Thanks @androolloyd, I'll check it later today!